src/dbt_osmosis/vendored/dbt_core_interface/project.py

```diff
@@ -601,9 +601,7 @@ def get_ref_node(self, target_model_name: str) -> "ManifestNode":
             ),
         )
 
-    def get_source_node(
```

Comment:
These changes are not related this pull request. To avoid conflict with https://github.com/z3z1ma/dbt-osmosis/pull/105, I ran linters.